integration_tests/tests/transaction-view-athena-tests.ts

```diff
@@ -0,0 +1,73 @@
+import { resourcePrefix } from "../helpers/envHelper";
+import {
+  startQueryExecutionCommand,
+  queryObject,
+  formattedQueryResults,
+} from "../helpers/athenaHelper";
+
+import {
+  deleteS3Event,
+  generateTestEventsAndValidateEventExists,
+  waitForTrue,
+} from "../helpers/commonHelpers";
+import { publishSNS } from "../helpers/snsHelper";
+import { snsInvalidEventNamePayload } from "../payloads/snsEventPayload";
+
+const prefix = resourcePrefix();
+const databaseName = `${prefix}-calculations`;
+
+describe("\nExecute athena query to retrive transaction data\n", () => {
+  test("price retrived from billing_curated athena view query should match with expected calculated price for 2 events", async () => {
```

Comment:
Not essential but you could rework these 3 cases using `test.each`, here's an article about it that I reference whenever I need the syntax https://elfi-y.medium.com/reduce-boilerplate-test-code-with-jest-it-each-30a0eec9776d

Reply:
looks this is like Example (data table ) in cucumber if not essential may be one for future tests or can do part of refactoring